openmls/src/tree/sender_ratchet.rs

```diff
@@ -11,9 +11,6 @@ use crate::tree::{index::LeafIndex, secret_tree::*};
 use super::index::NodeIndex;
 use super::*;
 
-// TODO #265: This should disappear
```

Comment:
So this was meant to be #243 and not #265?

Reply:
I guess I didn't predict the future correctly. It wasn't clear which issue would be implemented first (or whether they would both be implemented together).

Reply:
Fair enough 😄